### PR TITLE
Feature/update changelog link

### PR DESCRIPTION
**Pull request by sughosneo**
## Summary

- Added changelog link for the following eBooks :

    - microservice
    - containerized-lifecycle
    - cloud-native

- Removed minor version number from the edition.

### CODE REVIEW

File: docs/architecture/cloud-native/index.md [hunk 1]
```diff
@@ -2,14 +2,16 @@
...
-ms.date: 05/13/2020
+ms.date: 10/11/2020
 ---
 
 # Architecting Cloud Native .NET Applications for Azure
 
 ![cover image](./media/cover.png)
 
-**EDITION v.1.0**
+**EDITION v.1**
```
Comment (nishanil) [inline docs/architecture/cloud-native/index.md:12]:
```suggestion
**EDITION v.1.0**
```

File: docs/architecture/cloud-native/index.md [hunk 2]
```diff
@@ -2,14 +2,16 @@
 title: Architecting Cloud Native .NET Applications for Azure
 description: A guide for building cloud-native applications leveraging containers, microservices, and serverless features of Azure.
 author: ardalis
-ms.date: 05/13/2020
+ms.date: 10/11/2020
```
Comment (nishanil) [inline docs/architecture/cloud-native/index.md:5]: dates need to be mm/dd/yyyy

**Review by nishanil — COMMENTED** [review]
Left a few comments.

File: docs/architecture/cloud-native/index.md (hunk 2 above)
Comment (sughosneo) [reply docs/architecture/cloud-native/index.md]: Yes, correct. Thanks for pointing it out.

**Review by nishanil — APPROVED** [review]

**Pull request merged**